src/main/java/brandkon/brand/BrandRestController.java

```diff
@@ -0,0 +1,31 @@
+package brandkon.brand;
+
+import brandkon.brand.DTO.BrandResponseDto;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class BrandRestController {
+
+    private BrandService brandService;
```

Comment:
```suggestion
    private final BrandService brandService;
```
`final`을 붙여 주면 더 좋습니다